CHANGELOG.md

```diff
@@ -48,6 +48,22 @@ Please top up your cycles balance by converting ICP to cycles like below:
 'dfx cycles convert --amount=0.123'.
 ```
 
+### chore: improve `dfx cycles convert` messages.
+
+If users run `dfx cycles convert` without enough ICP tokens, show additional messages to indicate what to do next.
+```
+Error explanation:
+Insufficient ICP balance to finish the transfer transaction.
+How to resolve the error:
+Please top up your ICP balance.
+
+Please run 'dfx ledger account-id' to get your account address for receiving ICP from centralized exchanges.
+(Example: a9e20b8d042cb7b73144de94e9091e8f2620dfc1e617cbfaa19d624322451b9d)
+
+Please run 'dfx identity get-principal' to get your principal for receiving ICP from ICP wallets and decentralized exchanges.
+(Exmaple: treln-6wg26-lak4v-sh7p6-qm7hn-ohcjl-ut3os-lemuf-ypxvr-zcamm-gae)
```

Comment:
This no longer matches the expected output

Reply:
Oops, fixed at https://github.com/dfinity/sdk/pull/4019/commits/fe88fecd4604f79b10f37824f4508d0028367aa3